website/docusaurus.config.ts

```diff
@@ -172,6 +173,7 @@ const config: Config = {
       ],
       copyright: `Copyright © ${new Date().getFullYear()} Privacy and Scaling Explorations`,
     },
+    zoom: {},
```

Comment:
Do we need to add selectors and other configuration here?

Reply:
yes, I tried to remove it and it will cause the package not working

Reply:
Here I mean the zoom configuration like it described in package readme.

```
zoom: {
      selector: '.markdown :not(em) > img',
      background: {
        light: 'rgb(255, 255, 255)',
        dark: 'rgb(50, 50, 50)'
      },
      config: {
        // options you can specify via https://github.com/francoischalifour/medium-zoom#usage
      }
    }
```

Do we need to configure it somehow or it works fine without it?

Reply:
Sorry for late replay
We don't need to add those config since they are default values